hca/dss/__init__.py

```diff
@@ -113,65 +113,66 @@ def download(self, bundle_uuid, replica, version="", dest_name="",
             retries_left = num_retries
             hasher = hashlib.sha256()
             with open(file_path, "wb") as fh:
-                while True:
-                    try:
-                        response = self.get_file._request(
-                            dict(uuid=file_uuid, version=file_version, replica=replica),
-                            stream=True,
-                            headers={
-                                'Range': "bytes={}-".format(fh.tell())
-                            },
-                        )
+                if file_['size'] > 0:
```

Comment:
nit: this code is getting too deeply nested. I'd change the flow control here such that it's flatter.  Example below.

```python
def download(self, ...
    ...
    for file_ in files.values():
        ...
        if file_['size'] > 0:
            _download_file(file_uuid, file_version, replica)
        ....

def _download_file(file_uuid, file_version, replica)
...
```

Reply:
Updated w implementation that doesn't nest another layer, but agree the function can be refactored/broken down. Choosing not to refactor now in favor of focusing on other Pilot Phase tickets. Thoughts?